en/7-iterate-improve-frequently.md

```diff
@@ -5,23 +5,6 @@ lang: en
 altLang: fr
 altLangPage: 7-effectuer-iterations-ameliorations-constamment
 ---
-<div class="dpgn-section-intro-standard">
```

Comment:
The divs and sections with class="dpgn-section-*" shouldn't be removed as they are used as landmarks for the automated dataset generation process. The content within them can be changed but the tags themselves should be left as is.